Makefile

```diff
@@ -47,10 +47,10 @@ flow:
 	./node_modules/.bin/flow check --strip-root
 
 lint:
-	./node_modules/.bin/eslint scripts $(SOURCES) '*.js' '**/.*.js' --format=codeframe --rulesdir="./scripts/eslint_rules"
```

Comment:
Why where these removed? I think that it was used to lint things like `babel.config.js`

Reply:
I tried to explain it above:
- removed non-existing glob:  '**/.*.js'   https://eslint.org/docs/5.0.0/user-guide/migrating-to-5.0.0#-linting-nonexistent-files-from-the-command-line-is-now-a-fatal-error
- in addition removed default file .js extension glob: '*.js'

Reply:
oh I see. just looking at it a little closer:  `'**/.*.js'`

I suppose there are no files with .something.js anymore. I guess babel.config had a dot prior.

eslint doesn't find anything, and throws.

Reply:
I think we could take out $(SOURCES), and replace it with '**/*.js' (everything), and .eslintignore excludes everything which needs to be excluded.

Reply:
Oh, i didn't notice the `.` in `.*.js`. It is ok to have sources instead of `**/*.js` (we would lint too much otherwise), but `*.js` should still be there. I tested it locally ant with this patch `babel.config.js` isn't linted anymore.

Reply:
I didn't really see it either :smiley: I only saw *eslint* complaining, and removed it.

alrighty, I'll put `'*.js'` back shortly and leave $(SOURCES) as is.